docs/src/operate/c8y/supported-operations.md

```diff
@@ -422,3 +476,4 @@ Available only in non-template operation file definition:
 Available only in the template operation file definition:
 * `workflow.operation` - The command name that will trigger workflow execution.
 * `workflow.input` - The JSON object input that can be used in the workflow.
+* `workflow.output` - The JSON object that is converted to the additinal parameters of the SmartREST successful status update messages (`501`/`504`)
```

Comment:
Fixing minor type (not picked up by the typo checker unfortunately)

```suggestion
* `workflow.output` - The JSON object that is converted to the additional parameters of the SmartREST successful status update messages (`501`/`504`)
```

Though are you sure the description is correct, doesn't the `workflow.output` get included in the 503/506 messages (e.g. setting the operation to `SUCCESSFUL` status and setting the result).

Reply:
True, 503/506 messages :D

Reply:
fixed in [14aa737](https://github.com/thin-edge/thin-edge.io/pull/3294/commits/14aa737582ad996088c50bd582dd50e372c7f811)